app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/returns/RepresenteeAnswers.scala

```diff
@@ -16,25 +16,29 @@
 
 package uk.gov.hmrc.cgtpropertydisposalsfrontend.models.returns
 
-import julienrf.json.derived
+import java.time.LocalDate
 
+import julienrf.json.derived
 import play.api.libs.json.OFormat
 import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.name.IndividualName
 
 sealed trait RepresenteeAnswers extends Product with Serializable
 
 object RepresenteeAnswers {
 
-  final case class IncompleteRepresenteeAnswers(name: Option[IndividualName], id: Option[RepresenteeReferenceId])
-      extends RepresenteeAnswers
+  final case class IncompleteRepresenteeAnswers(
+    name: Option[IndividualName],
+    id: Option[RepresenteeReferenceId],
+    dateOfDeath: Option[LocalDate]
```

Comment:
could we create a wrapper around `LocalDate` like this:
```scala
final case class DateOfDeath(value: LocalDate) extends AnyVal
```
- when we're using it elsewhere it'd make it clearer in the types what's going on